changelogs/fragments/ipsubnet_performance.yaml

```diff
@@ -0,0 +1,10 @@
+---
+bugfixes:
```

Comment:
This will be minor_change as you are updating type of arg from str to int

Reply:
@ashwini-mhatre I've split out the changes so the performance work is under bugfix and the parameter change is under `minor_changes`.